src/extension.ts

```diff
@@ -738,6 +738,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     }
   }
 
+  // The URI's of all classes that have been opened. Used when objectscript.openClassContracted is true.
```

Comment:
Nitpick - remove apostrophe:

```suggestion
  // The URIs of all classes that have been opened. Used when objectscript.openClassContracted is true.
```